### DISCUSSION TITLE
Quick questions about diagnostics messages

### DISCUSSION THREAD

**Discussion by LandonSchropp** [LazyVim/LazyVim#377] [Q&A]
Hello again. 👋 

When I'm editing files, I see a lot of these little diagnostics notifications.

<img width="425" alt="Screen Shot 2023-03-03 at 11 57 20 AM" src="https://user-images.githubusercontent.com/361591/222814941-a07676bd-a084-4519-84cd-4eb4bc9a87a7.png">

A few quick questions:

* Which plugin is causing these to pop up? Is it Noice?
* Can I disable these notifications for diagnostics only? (The formatting ones are handy for Ruby.)
* If the last answer is no, can I disable these notifications entirely?

Thanks!

**Comment by folke** [comment]
You can disable them. Check the noice docs to disable lsp progress

**Reply by luxus** [reply]
i guess he is wondering why there are so many, at least i do..

**Reply by folke** [reply]
Servers manage those messages, so you would need to check the lsp docs.

**Reply by LandonSchropp** [reply]
Gotcha, thanks!
